static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -139,46 +235,61 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
           ))}
         />
-        <h5 className="u-no-margin--bottom">Networks</h5>
+        <hr className="u-spaced-hr" />
+        <h5 className="p-text--x-small-capitalised u-no-margin--bottom">
+          Networks
+        </h5>
         <List
           className="u-no-margin--bottom"
           items={Object.values(instance.expanded_devices)
             .filter(isNicDevice)
-            .map((item) =>
-              item.name ? `${item.name}: ${item.network}` : item.network
-            )}
+            .map((item) => (
+              // TODO: fix this link to point to the network detail page
+              <Link key={item.network} to={`/ui/${instance.project}/networks`}>
+                {item.network}
+              </Link>
+            ))}
         />
+        <hr className="u-spaced-hr" />
         <Row>
-          <Col size={1}>
-            <h5 className="u-no-margin--bottom">Snapshots</h5>
+          <Col size={3}>
+            <h5 className="p-text--x-small-capitalised u-no-margin--bottom">
+              Recent Snapshots
+            </h5>
           </Col>
-          <Col
-            size={3}
-            className="p-snapshot-link p-text--small u-align--right u-no-margin--bottom"
-          >
-            <Link
-              to={`/ui/${instance.project}/instances/${instance.name}/snapshots`}
+          {instance.snapshots?.length && (
+            <Col
+              size={1}
+              className="p-snapshot-link p-text--small u-align--right u-no-margin--bottom u-truncate"
             >
-              Manage snapshots
-            </Link>
-          </Col>
+              <strong>
+                <Link
+                  to={`/ui/${instance.project}/instances/${instance.name}/snapshots`}
+                >
+                  View all
+                </Link>
+              </strong>
+            </Col>
+          )}
         </Row>
         <List
           className="u-no-margin--bottom"
           items={
-            instance.snapshots?.map((snapshot) => (
-              <Row key={snapshot.name}>
-                <Col size={1} className="u-truncate" title={snapshot.name}>
-                  {snapshot.name}
-                </Col>
-                <Col
-                  size={3}
-                  className="p-snapshot-creation p-text--small u-align--right u-text--muted u-no-margin--bottom"
-                >
-                  created {isoTimeToString(snapshot.created_at)}
-                </Col>
-              </Row>
-            )) ?? ["None"]
+            instance.snapshots
+              ?.slice(0, RECENT_SNAPSHOT_LIMIT)
+              .map((snapshot) => (
+                <Row key={snapshot.name}>
+                  <Col size={2} className="u-truncate" title={snapshot.name}>
+                    {snapshot.name}
+                  </Col>
+                  <Col
+                    size={2}
+                    className="p-snapshot-creation p-text--small u-align--right u-text--muted u-no-margin--bottom"
```

Comment:
This looks a little odd, because the text is smaller - I think in the design it is normal size but italics.